src/docs/product/performance/filters-display.mdx

```diff
@@ -18,19 +18,19 @@ Each tab in **Performance** displays an automatically selected set of [widgets](
 
 ### All Transactions
 
-When you select "All Transactions", the table includes all frontend and backend transactions.
+When you select "All Transactions", the table includes all transactions across any projects and period you have selected.
 
 <!-- ![Performance Homepage with Display: All](00_performance_landing.png) -->
 
 ### Web Vitals {#frontend-pageload}
 
-[Web Vitals](/product/performance/web-vitals/) includes frontend pageload transactions. The transactions are automatically filtered to only include `transaction.op:pageload`. This generally corresponds to all browser page transitions that are not handled internally in SPA frameworks, such as React or Vue.
+[Web Vitals](/product/performance/web-vitals/) includes frontend pageload transactions. The transactions are automatically filtered to only include `transaction.op:pageload`. This generally corresponds to all browser page transitions that are the first load of a page, and include performance information from the beginning of the load itself as well as performance of the SPA frameworks being loaded, such as React or Vue, if available.
```

Comment:
This isn't quite right now that I read it, @doralchan any ideas on how we explain Web Vitals vs. Frontend better?